packages/graph-explorer/src/connector/openCypher/templates/oneHopTemplate.ts

```diff
@@ -107,7 +107,6 @@ const oneHopTemplate = ({
   edgeTypes = [],
   filterCriteria = [],
   limit = 10,
-  offset = 0,
```

Comment:
We should remove the same from `oneHopTemplate`'s docstring:

https://github.com/aws/graph-explorer/blob/122169c546276f5234dd1990a6493bf46854ed27/packages/graph-explorer/src/connector/openCypher/templates/oneHopTemplateEdges.ts#L95

Reply:
Good catch

Reply:
Fixed